library/templates/address_list_form.php

```diff
@@ -0,0 +1,278 @@
+        let state = row_form_addresses.querySelector("input.form_addresses_state").value;
+        let zip = row_form_addresses.querySelector("input.form_addresses_zip").value;
+        let plus_four = row_form_addresses.querySelector("input.form_addresses_plus_four").value;
+        let full_zip = zip + ((isBlank(zip) || isBlank(plus_four)) ? "": "-" + plus_four);
+        let country = row_form_addresses.querySelector("input.form_addresses_country").value;
+
+ */
+$name_field_id = "form_" . $field_id_esc;
+?>
+<div>
+    <table class='form_addresses table table-sm' id="<?php echo attr($table_id); ?>">
+        <thead class ="thead-light">
+        <tr>
+            <th colspan="2">
+                <?php echo xlt("Additional Addresses"); ?>&nbsp;&nbsp;&nbsp;&nbsp;
+                <span class="fas fa-plus-square" style="color:#007bff;" onclick="AddAddress(event);return false"></span>
+            </th>
+        </tr>
+        </thead>
+    </table>
+
+</div>
+
+<template class="template_add_address">
+    <tr class='display_addresses'>
+        <td class='noPrint col-1'>
+            <span class="fas fa-fw fa-edit text-primary" onclick="EditAddress(this);return false"></span>
+            <span class="fas fa-fw fa-trash-alt text-danger" onclick="DeleteAddress(this);return false"></span>
+        </td>
+        <td class='display_addresses_full_address'></td>
+
+    </tr>
+
+    <tr class='d-none form_addresses'>
+        <input type="hidden" class="form_addresses_data_action" name="<?php echo attr($name_field_id); ?>[data_action][]" value="<?php echo xla("No Change"); ?>" />
+        <input type="hidden" class="form_addresses_id" name="<?php echo attr($name_field_id); ?>[id][]" value="" />
+        <input type="hidden" class="form_addresses_foreign_id" name="<?php echo attr($name_field_id); ?>[foreign_id][]" value="" />
+        <td></td>
+        <td colspan="1">
+            <table class="table table-borderless"><tr>
+                <tr>
+                    <td width='200' colspan='3'><?php echo xlt("Street Line 1"); ?></td>
+                    <td width='200' colspan='2'><?php echo xlt("Street Line 2"); ?></td>
+                </tr>
+                <tr>
+                    <td colspan='3'><input type='text' class="form_addresses_line1" name="<?php echo attr($name_field_id); ?>[line_1][]" style='width:200px;' value='' tabindex='4'></td>
+                    <td colspan='1'><input type='text' class="form_addresses_line2" name="<?php echo attr($name_field_id); ?>[line_2][]" style='width:200px;' value='' tabindex='4'></td>
+
+                <tr height='10'><td></td></tr>
+                <tr>
+                    <td width='50' colspan='1'><?php echo xlt("City"); ?></td>
+                    <td width='8' colspan='1'><?php echo xlt("State"); ?></td>
+                    <td width='12' colspan='1'><?php echo xlt("Postal Code"); ?></td>
+                    <td width='12' colspan='1'><?php echo xlt("Plus Four"); ?></td>
```

Comment:
remove the Plus Four and instead store everything in Postal Code

Reply:
Made everything use Postal Code on the frontend.  I did put in logic to split the postal code into zip + plus_four if the country is USA when we are saving to the database.  Seems like that'd be a good thing for data searching, but not sure if that will break in other parts of the system.  If its an issue I can just store the data directly into postalcode.

Reply:
would just use postalcode since the plus_four is never used in OpenEMR at this time (at least I couldn't find a place it is used). then never need to have country specific logic.